### PR TITLE
[simplebar] add missing `unMount` instance method

### PR DESCRIPTION
**Pull request by breadadams**
Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [X] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/Grsmto/simplebar/blob/master/packages/simplebar/src/simplebar.js#L906-L912

### CODE REVIEW

**Review by GregOnNet — APPROVED** [review]

File: types/simplebar/index.d.ts [hunk 1]
```diff
@@ -16,6 +19,7 @@ declare class SimpleBar {
     recalculate(): void;
     getScrollElement(): Element;
     getContentElement(): Element;
+    unMount(): void;
```
[the remark this answers is not in the document]
Comment (breadadams) [reply types/simplebar/index.d.ts:22]: Thanks, I'll adjust it this afternoon 👍

[the remark this answers is not in the document]
Comment (breadadams) [reply types/simplebar/index.d.ts:22]: @GregOnNet I'm not sure what whitespace you're referring to here, there are none as far as I can see. I addressed the empty line in the `simplebar-tests` though.

[the remark this answers is not in the document]
Comment (breadadams) [reply types/simplebar/index.d.ts:22]: @GregOnNet ping 😇

**Review by GregOnNet — APPROVED** [review]

**Comment by breadadams** [comment]
Ready to merge

**Pull request merged**